bigtable-hbase-mirroring-client-1.x-parent/bigtable-hbase-mirroring-client-1.x/src/main/java/com/google/cloud/bigtable/mirroring/hbase1_x/utils/flowcontrol/RequestCountingFlowControlStrategy.java

```diff
@@ -85,9 +83,13 @@ public boolean canAcquireResource(RequestResourcesDescription requestResourcesDe
     }
 
     @Override
-    public void accountAcquiredResource(RequestResourcesDescription requestResourcesDescription) {
-      this.primaryReadsAdvantage += requestResourcesDescription.numberOfResults;
-      this.usedBytes += requestResourcesDescription.sizeInBytes;
+    public boolean tryAcquiredResource(RequestResourcesDescription requestResourcesDescription) {
```

Comment:
nit: I think you wanted to call it `tryAcquireResource` (without the extra "`d`")

Reply:
Done.